static/app/views/settings/projectAlerts/issueRuleEditor/index.tsx

```diff
@@ -518,7 +519,13 @@ class IssueRuleEditor extends AsyncView<Props, State> {
 
     const userTeams = new Set(teams.filter(({isMember}) => isMember).map(({id}) => id));
```

Comment:
would prefer this not be a set anymore since its only used in one place
```suggestion
    const userTeams = teams.filter(({isMember}) => isMember).map(({id}) => id);
```

Reply:
or reuse this set somehow

Reply:
makes sense, I'll get rid of the sets for issue alerts and metric alerts.